schema.graphql

```diff
@@ -206,10 +220,25 @@ type Event @entity {
   amount: BigDecimal!
   "Block timestamp"
   blockTime: Int!
+  "Block number"
+  blockNumber: Int!
   "Amount of Tokens transferred - used only in specific cases"
   xTokenAmount: BigDecimal
   "Liquidator that paid the debt: Relevant only for Liquidation event"
   liquidator: Account
   "Account of the payer - Related to Repay event"
   payer: Account
+  "Account that was paid by account field"
+  onBehalfOf: Account
+  "Interest rate mode"
+  interestRateMode: Int
+  "Borrow rate"
+  borrowRate: BigInt
+  "Code used to register the integrator originating the operation, for potential rewards"
+  referralCode: Int
```

Comment:
I think referral code is unlikely to be used by us for a very long time, if ever. I think we can remove it